xsuite/src/data/encoding.ts

```diff
@@ -251,6 +251,16 @@ export const e = {
     }
     return account as PreserveDefinedness<T, Account>;
   },
+  MapperKey: (...args: EncodableMapperKeyArgs) => {
```

Comment:
Let's move this code right after all the encoders that start with uppercase (`e.Option`) as it is the same kind.